compliance_checker/tests/test_cf.py

```diff
@@ -1296,3 +1296,30 @@ def test_cell_measures(self):
         score, out_of, messages = self.get_results(results)
         message = u'Cell measure variable box_area referred to by PS is not present in dataset variables'
         assert message in messages
+
+    def test_check_conventions_are_cf_16(self):
```

Comment:
I'm probably going to remove the inheritance from `TestCF1_6` as it largely ends up being a bunch of repeated tests.

Reply:
Without the inheritance, though,  wont't we lose most of the existing checks that `CF-1.7` *does* use?

Reply:
Yes, but they're largely redundant because they will have the same behavior.  For those that don't we can just call the extended method and test for behavior specific to that checker subclass.